specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/NfsV3AccountCreate.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res9101",
+    "accountName": "sto4445",
+    "api-version": "2020-08-01-preview",
+    "parameters": {
+      "sku": {
+        "name": "Premium_LRS"
+      },
+      "kind": "BlockBlobStorage",
+      "location": "eastus",
+      "properties": {
+        "isHnsEnabled": true,
+        "isNfsV3Enabled": true,
+        "supportsHttpsTrafficOnly": false,
+        "networkAcls": {
```

Comment:
please use the right type for networkAcls. It should be array.

Reply:
For more information, see ModelValidation failures.

Reply:
Please use the valid value in Bypass/defaultaction,Virtualnetworkrule. The following value are not valid, like default action should be Deny or Allow

Reply:
fixed